infra/build_specified_commit.py

```diff
@@ -231,7 +231,7 @@ def cleanup():
                                        env_to_add=None,
                                        source_path=host_src_path,
                                        mount_location='/src')
-    if result == 0 or i == num_retry:
+    if result or i == num_retry:
```

Comment:
I think you need to fix line 288 as well. Please double check other usages that need to be changed as well.

Reply:
Done.